src/hooks/use-axios.js

```diff
@@ -1,46 +1,61 @@
-import { useState, useEffect, useCallback } from 'react'
-import axios from 'axios'
+import { useEffect, useState } from 'react'
+
+import { useMutation, useQueryClient } from '@tanstack/react-query'
 
-const DELAY = 1 * 1000 // Artificial delay in ms
+import axios from 'axios'
 
-function useAxios(run = false, urlRun, bodyRun, methodRun, headersRun) {
+export default function useAxios(
+  run = false,
+  keys = ['key'],
+  urlRun,
+  bodyRun,
+  methodRun,
+  headersRun
```

Comment:
These don't appear to be used anywhere. Are they needed for some reason I'm missing?

Reply:
We will use them in the future ... if that's ok . .can i keep them ?